### DISCUSSION TITLE
Need help with anything?

### DISCUSSION THREAD

**Discussion by vghfr** [zer0Kerbal/MoarKerbals#75] [General]
I've never modded ksp, but I am good at c#, can't be that hard to help out 😉

**Comment by zer0Kerbal** [comment]
let me get the next release out so can clear the air in the repo - :)

**Reply by vghfr** [reply]
👍

**Comment by zer0Kerbal** [comment]
the C# is straight forward for the most part. Limited to .NET 4.7.2 (because that is what the game is using), but I use the new features of C# 10.0 with out any issues.

The issues come with Unity and KSP's API. located [here](https://www.kerbalspaceprogram.com/api/index.html).

The real issues then come with interacting with other mods.

**Reply by vghfr** [reply]
Yeah, I wasn't expecting KSP's API to be something that's... Dare I say, cooperative at times.

**Comment by zer0Kerbal** [comment]
Once you know where something is handled - VSCommunity's peek really helps.
